packages/mui-joy/src/Typography/Typography.test.tsx

```diff
@@ -101,8 +103,8 @@ describe('<Typography />', () => {
           <Typography>Foo</Typography> - <Typography component="b">Bar</Typography>
         </Typography>,
       );
-      expect(container.firstChild.firstChild.tagName).to.equal('SPAN');
-      expect(container.firstChild.lastChild.tagName).to.equal('B');
+      expect((container.firstChild?.firstChild as HTMLElement)?.tagName).to.equal('SPAN');
+      expect((container.firstChild?.lastChild as HTMLElement)?.tagName).to.equal('B');
```

Comment:
```suggestion
      const child = container.firstChild?.firstChild ?? null;
      if (child === null) {
        return;
      }
      expect((child as HTMLElement).tagName).to.equal('SPAN');
      expect((child as HTMLElement).tagName).to.equal('B');
```

Reply:
I'm not sure if this change would be appropiate:
The first `expect` checks for the `firstChild` while the second checks for the `lastChild`. I'll be changing this so that I put the `container.firstChild` in a value and check against `null`

Reply:
Oh, I missed that. Yes I exactly meant what you've done :) THanks